src/api/app/controllers/staging/workflows_controller.rb

```diff
@@ -0,0 +1,47 @@
+class Staging::WorkflowsController < ApplicationController
+  before_action :require_login
+  before_action :set_project
+
+  def create
+    staging_workflow = @project.build_staging
+    authorize staging_workflow
+
+    staging_workflow.managers_group = Group.find_by(title: xml_hash)
+
+    unless staging_workflow.managers_group
+      render_error(
+        status: 404,
+        errorcode: 'not_found',
+        message: "Managers Group '#{params[:managers_title]}' not found."
+      )
+      return
+    end
```

Comment:
You can do this, as you do in other controllers, in a `before_filter`.

Reply:
I kept the same structure as in the WebUI code. There it is not in a `before_filter`

Reply:
That's not a reason. Doing the right[tm] thing trumps consistency.

Reply:
@hennevogel 

> That's not a reason. Doing the right[tm] thing trumps consistency.

well... that moving this to a filter is the right thing is questionable. You can use a `before_filter` for this, but there is also not reason to use a `before_filter`. I am not repeating code or writing a huge method so that this seems to be needed. That's why I though that being consistent make sense in this case. But I can also change it...

Reply:
Now that I am raising an exception as @coolo suggested, I don't think this is an issue anymore. Is it?